package.json

```diff
@@ -17,23 +17,23 @@
         "registry": "https://registry.npmjs.org"
     },
     "dependencies": {
-        "@mojaloop/central-services-logger": "8.1.2",
+        "@hapi/hapi": "18.4.1",
+        "@mojaloop/central-services-logger": "10.4.0",
         "boom": "7.3.0",
-        "@hapi/hapi": "18.4.0",
         "hapi-openapi": "1.2.4"
```

Comment:
hapi-openapi can be updated to the latest version and so can hapi. this should get rid of your audit issues